Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER hello@codeclimate.com
 
 RUN adduser -u 9000 -D app
+
+WORKDIR /code
+
+COPY build/ /usr/src/app
+COPY build/codeclimate-gofmt /usr/src/app/build/codeclimate-gofmt
```

Comment:
If you've copied all of `build/` into `/usr/src/app`, that ahas already put `/usr/src/app/codeclimate-gofmt` in place. Why not just use that, instead of making another copy in a `build` subdirectory?